packages/parser-typescript-config/README.md

```diff
@@ -0,0 +1,58 @@
+# Parser typescript-config (`@sonarwhal/parser-typescript-config`)
+
+The `typescript-config` parser allows the user to analyze the
+TypeScript configuration in their projects.
+
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @sonarwhal/parser-typescript-config
```

Comment:
`--save-dev` ?

Reply:
I don't think so. you have to install the package to use the parser.

Reply:
`sonarwhal` and related package should be used as part of the development process, so they should be used as dev dependencies not dependencies. 

Doing

> npm install @sonarwhal/parser-typescript-config

will install `@sonarwhal/parser-typescript-config` as a dependency not a devDependency, because  ` npm install @sonarwhal/parser-typescript-config` is the same as ` npm install --save @sonarwhal/parser-typescript-config`

From https://docs.npmjs.com/cli/install#description:

> By default, npm install will install all modules listed as dependencies in package.json.

Reply:
Maybe the confusion is because I missed `-g` but if you have sonarwhal installed globally and you want to use this parser, you have to install the package normally, not as a dev dependency.